build.gradle.kts

```diff
@@ -35,9 +35,3 @@ allprojects {
         }
     }
 }
-
-// Workaround for https://youtrack.jetbrains.com/issue/KT-49109 that allows building on Apple Silicon
-// This should no longer be needed with Kotlin version 1.6.20
-rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin> {
-    rootProject.the<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension>().nodeVersion = "16.0.0"
-}
```

Comment:
Node.js 16.x.x should be default for Kotlin 1.6.20, so this workaround should no longer be needed.